sdcm/utils/nemesis.py

```diff
@@ -113,8 +114,8 @@ def create_job_files_from_template(self) -> list[Path]:
             additional_configs = config.NEMESIS_REQUIRED_ADDITIONAL_CONFIGS.get(cls, [])
             additional_params = config.NEMESIS_ADDITIONAL_PIPELINE_PARAMS.get(cls, {})
             config_name = [
-                str(self.nemesis_test_config_dir / f"{self.base_job}-nemesis.yaml"),
```

Comment:
Why are you switching the order ?

Reply:
This is just to keep consistency in test resources naming.

Now all dedicated nemesis configs (the configurations/nemesis/*Monkey.yaml ones) have user prefix defined as 
e.g. 'user_prefix: 'longevity-5gb-1h-CorruptThenRepairMonkey''.
And if a user runs a docker backend on Jenkins, and sees that job uses configurations/nemesis/longevity-1gb-1h-nemesis.yaml base config, and in log sees that prepare stress command writes only 1G of data, but then sees that test resources are named as 'longevity-5gb-*-db-node-*' - it can cause confusion.

*Monkey.yaml configs are changed rarely (if ever). So when the 'longevity-1gb-1h-' (or similar) config is placed last in the list of configs for Job, we overwrite the user_prefix parameter value to exactly what the test does.

Reply:
Reverting this as well. The custom user_prefix will come from docker_backend_local config, as we decided not to multiply base configs for Job generator.